plugins/action/common/prepare_plugins/prep_107_vrf_lites.py

```diff
@@ -234,7 +234,7 @@ def prepare(self):
                         #                   ebgp_distance: 25
                         #                   ibgp_distance: 180
                         #                   local_distance: 200
-                        if "bgp" in vrf_lite:
+                        if "bgp" in vrf_lite and vrf_lite["bgp"] is not None:
```

Comment:
Should be able to simply this to (but please test)

```python
if vrf_lite.get("bgp") is not None:
```

Reply:
I tested and the code you suggested works, I pushed the update. Thanks.

Reply:
Will merge as soon as the tests complete/pass